src/components/tile-select/tile-select.test.js

```diff
@@ -0,0 +1,641 @@
+            type="info"
+            tooltipMessage="This tile cannot be removed at this time"
+          />
+        }
+        {...props}
+      />
+    </TileSelectGroup>
+  );
+};
+const TileSelectGroupComponent = ({ ...props }) => {
+  const [value, setValue] = React.useState(null);
+  return (
+    <TileSelectGroup
+      name="Tile Select"
+      value={value}
+      legend="Tile Select"
+      description="Pick one of the available options"
+      checked={value}
+      onChange={(e) => setValue(e.target.value)}
+      {...props}
+    >
+      <TileSelect
+        value="1"
+        id="1"
+        aria-label="1"
+        title="Title"
+        subtitle="Subtitle"
+        description="Short and descriptive description"
+        checked={value}
+        onChange={(e) => setValue(e.target.value)}
+      />
+      <TileSelect
+        value="2"
+        id="2"
+        aria-label="2"
+        title="Title1"
+        subtitle="Subtitle1"
+        description="Short and descriptive description1"
+        checked={value}
+        onChange={(e) => setValue(e.target.value)}
+      />
+    </TileSelectGroup>
+  );
+};
+
+context("Tests for TileSelect component", () => {
+  describe("check props for TileSelect component", () => {
+    it.each([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS])(
+      "should check title as %s for TileSelect component",
+      (title) => {
+        CypressMountWithProviders(<TileSelectComponent title={title} />);
+        titleElement().should("have.text", title);
+      }
+    );
+
+    it.each(["add", "info"])(
+      "should check titleAdornment iconType as %s for TileSelect component",
+      (iconType) => {
+        CypressMountWithProviders(
+          <TileSelectComponent titleAdornment={<Icon type={iconType} />} />
+        );
+        getComponent("icon")
+          .should("have.attr", "type", iconType)
+          .and("be.visible");
+      }
+    );
+
+    it.each([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS])(
+      "should check subtitle as %s for TileSelect component",
+      (subtitle) => {
+        CypressMountWithProviders(<TileSelectComponent subtitle={subtitle} />);
+        subtitleElement().should("have.text", subtitle);
+      }
+    );
+
+    it.each([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS])(
+      "should check description as %s for TileSelect component",
+      (description) => {
+        CypressMountWithProviders(
+          <TileSelectComponent description={description} />
+        );
+        descElement().should("have.text", description);
+      }
+    );
+
+    it.each([
+      [false, "rgba(0, 0, 0, 0.9)"],
+      [true, "rgba(0, 0, 0, 0.3)"],
+    ])(
+      "should check when TileSelect component set as %s",
+      (disabled, borderColor) => {
+        CypressMountWithProviders(<TileSelectComponent disabled={disabled} />);
+        inputElement().should("have.css", "border-color", borderColor);
+      }
+    );
+
+    it.each([1, 2, 3, 4])(
```

Comment:
The `value` prop accepts only a `string`

```suggestion
    it.each(["1", "2"])(
```